ReactAndroid/build.gradle

```diff
@@ -280,6 +280,7 @@ task androidSourcesJar(type: Jar) {
 
 android {
     compileSdkVersion 31
+    ndkVersion rootProject.ext.ndkVersion
```

Comment:
I was setting this too [earlier](https://github.com/reactwg/react-native-releases/discussions/13#discussioncomment-2269318) in Android Studio manually when integrating other libraries, since it wasn't being picked up from `rootProject`

So i've had a `yarn patch` stash sitting awhile locally, but [avoided committing it in the end](https://github.com/software-mansion/react-native-screens/issues/1389#issuecomment-1086895179) (could jus be me)

I don't see the harm including it, but assumed it wasn't for a reason and can't think of side-effects it would cause atm

Reply:
I think there needs to be such change made in order to make developer's life easier. Especially in early phases of development like starting with the template which already should have as I remember correctly some ndk version put there, but if developer wants to upgrade it to newer version manually then I see no problem there. Maybe it would be better for the ReactAndroid to update this ndk as internal dependency to support M1 platform, but still someone might want to use some version that he has already installed and if there is no problem with that as you mentioned, this change would be beneficial, because it would reduce digging through the deeps of the code to set it somewhere else.

Reply:
Gotcha and thanks the explanation earlier ^ appreciate it

Just back from a mini-break and spotted (thx too @cortinico)
- https://github.com/reactwg/react-native-releases/discussions/19#discussioncomment-2525259
- https://github.com/facebook/react-native/pull/33588

Which looks like it'll resolve our NDK issue too, I'll check later today